### DISCUSSION TITLE
Dynamic Block Properties Supported?

### DISCUSSION THREAD

**Discussion by warappa** [DomCR/ACadSharp#484] [Q&A]
With the very recently added support for dynamic blocks, is it possible to extract dynamic properties?
More specifically: I mean the data retrieved by `BlockReference.DynamicBlockReferencePropertyCollection` in AutoCAD's .NET API.

In my case I would already be satisfied if I get
- property name (`DynamicBlockReferenceProperty.PropertyName` in AutoCAD's .NET API)
- property description (`DynamicBlockReferenceProperty.Description` in AutoCAD's .NET API)
- property value (`DynamicBlockReferenceProperty.Value` in AutoCAD's .NET API)
- visibility (`DynamicBlockReferenceProperty.VisibleInCurrentVisibilityState` in AutoCAD's .NET API)

**Comment by DomCR** [comment]
Hi @warappa,

I've looking at dynamic block for a while, here is a PR that is meant to extract the information:

- https://github.com/DomCR/ACadSharp/pull/486

Is not yet implemented but I'm trying to get as much information as possible by now, I'll try to add the properties that you mentioned in this PR.
